src/sipp.cpp

```diff
@@ -756,7 +756,15 @@ static char* wrap(const char* in, int offset, int size)
 /* If stdout is a TTY, wrap stdout in a call to PAGER (generally less(1)).
  * Returns a pid_t you'll have to pass to end_pager(). */
 static pid_t begin_pager() {
-    char pager[15] = "/usr/bin/pager";
+    char const *pager_options[4] = {
```

Comment:
`std::vector<std::string>`? Iteration is easier, and you won't need a copy, you can pass `&opt[0]`.

Reply:
:shrug: 

I guess we should do more c++. Be my guest :grimacing: 

I assume we won't need a copy because it _is_ a copy already.